layouts/docs/list.html

```diff
@@ -1,5 +1,5 @@
-<!-- Modified from docsy theme to show related content -->
-<!-- https://github.com/google/docsy/blob/c65715725ee992cd4e59aeefe0f66df80b2eb4ea/layouts/docs/list.html -->
+{{/* Modified from docsy theme to show related content */}}
+{{/* https://github.com/google/docsy/blob/c65715725ee992cd4e59aeefe0f66df80b2eb4ea/layouts/docs/list.html */}}
```

Comment:
This is more evidence that this template was never actually getting used by Hugo at all.  Hugo can be picky about where you can use normal HTML comments and where you have to use these special template comments.  Once I moved this file and it started getting used, Hugo failed to load the page because of these bad comments...